web/.storybook/css-import-maps.ts

```diff
@@ -7,123 +8,70 @@
+    'import PFForm from "@patternfly/patternfly/components/Form/form.css";',
+    'import PFFormControl from "@patternfly/patternfly/components/FormControl/form-control.css";',
+    'import PFGallery from "@patternfly/patternfly/layouts/Gallery/gallery.css";',
+    'import PFGlobal from "@patternfly/patternfly/patternfly-base.css";',
+    'import PFGrid from "@patternfly/patternfly/layouts/Grid/grid.css";',
+    'import PFHint from "@patternfly/patternfly/components/Hint/hint.css";',
+    'import PFInputGroup from "@patternfly/patternfly/components/InputGroup/input-group.css";',
+    'import PFLabel from "@patternfly/patternfly/components/Label/label.css";',
+    'import PFList from "@patternfly/patternfly/components/List/list.css";',
+    'import PFLogin from "@patternfly/patternfly/components/Login/login.css";',
+    'import PFModalBox from "@patternfly/patternfly/components/ModalBox/modal-box.css";',
+    'import PFNav from "@patternfly/patternfly/components/Nav/nav.css";',
+    'import PFNotificationBadge from "@patternfly/patternfly/components/NotificationBadge/notification-badge.css";',
+    'import PFNotificationDrawer from "@patternfly/patternfly/components/NotificationDrawer/notification-drawer.css";',
+    'import PFPage from "@patternfly/patternfly/components/Page/page.css";',
+    'import PFPagination from "@patternfly/patternfly/components/Pagination/pagination.css";',
+    'import PFProgressStepper from "@patternfly/patternfly/components/ProgressStepper/progress-stepper.css";',
+    'import PFRadio from "@patternfly/patternfly/components/Radio/radio.css";',
+    'import PFSelect from "@patternfly/patternfly/components/Select/select.css";',
+    'import PFSidebar from "@patternfly/patternfly/components/Sidebar/sidebar.css";',
+    'import PFSizing from "@patternfly/patternfly/utilities/Sizing/sizing.css";',
+    'import PFSpacing from "@patternfly/patternfly/utilities/Spacing/spacing.css";',
+    'import PFSpinner from "@patternfly/patternfly/components/Spinner/spinner.css";',
+    'import PFStack from "@patternfly/patternfly/layouts/Stack/stack.css";',
+    'import PFSwitch from "@patternfly/patternfly/components/Switch/switch.css";',
+    'import PFTable from "@patternfly/patternfly/components/Table/table.css";',
+    'import PFTabs from "@patternfly/patternfly/components/Tabs/tabs.css";',
+    'import PFTitle from "@patternfly/patternfly/components/Title/title.css";',
+    'import PFToggleGroup from "@patternfly/patternfly/components/ToggleGroup/toggle-group.css";',
+    'import PFToolbar from "@patternfly/patternfly/components/Toolbar/toolbar.css";',
+    'import PFTreeView from "@patternfly/patternfly/components/TreeView/tree-view.css";',
+    'import PFWizard from "@patternfly/patternfly/components/Wizard/wizard.css";',
+    'import ThemeDark from "@goauthentik/common/styles/theme-dark.css";',
+    'import styles from "./LibraryPageImpl.css";',
+];
+
+const cssImportMaps = rawCssImportMaps.reduce((acc, line) => (
```

Comment:
No sense in recording what can be generated.  This is cheap and only happens in the context of Storybook, so performance (70 lines!) isn't a super big deal there.  Besides, this is more readable: "Here are the import strings.  Here's the single modification that makes them palatable to Storybook."